src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -75,6 +73,14 @@ public ExtensionsRunner() throws IOException {}
     private final Logger logger = LogManager.getLogger(ExtensionsRunner.class);
     private final TransportInterceptor NOOP_TRANSPORT_INTERCEPTOR = new TransportInterceptor() {
     };
+    private NamedWriteableRegistryApi namedWriteableRegistryApi = new NamedWriteableRegistryApi();
+
+    /**
+     * Constructor for ExtensionsRunner
```

Comment:
Any reason for the changing the javadocs here?

Reply:
Javadocs for the extension runner constructor was missing and it was failing the strict javadoc check

Reply:
But we had javadocs already https://github.com/opensearch-project/opensearch-sdk/pull/54/files#diff-8208658a7c4e8521b2d0ad827f1cb9c9890f66eb568e8dce05215507d3d80911L64 :slightly_smiling_face: